contrib/ansible/openshift/README.md

```diff
@@ -33,11 +33,25 @@ Your inventory should have groups of hosts describing the roles they play in the
 [glusterfs]
```

Comment:
Is this a misnomer? Should it not be `cns` instead? Not sure whether you can add it as a synonym and then deprecate glusterfs.